models/User.js

```diff
@@ -32,12 +32,16 @@ User.init(
             unique: true,
             allowNull: false,
         },
+        bio: {
+            type: DataTypes.STRING,
```

Comment:
@isayahdurst - I suggest to changes datatype to DataTypes.TEXT
In thsi case no need validate legth and also user can add all information he/she wants